be/compendium/db/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
```

Comment:
no need to create another `url.py` here. utilize the one inside the `compendium` folder